conftest.py

```diff
@@ -176,6 +177,17 @@ def xfail_on_xcpng_8_3(host, request):
     if host.xcp_version >= version.parse("8.3"):
         request.node.add_marker(pytest.mark.xfail)
 
+@pytest.fixture(scope='session')
+def host_no_ipv6(host):
+    if is_ipv6(host.hostname_or_ip):
+        pytest.skip(f"This test requires an IPv4 XCP-ng")
+
+@pytest.fixture(scope='function')
+def xfail_on_xcpng_ipv6(host, request):
```

Comment:
We don't need this fixture anymore, right?

Reply:
Right, I thought about remove and then thought it might be handy in the future.
I can remove it if you'd like.